crates/noirc_frontend/src/hir_def/types.rs

```diff
@@ -124,6 +129,39 @@ pub struct StructType {
     pub span: Span,
 }
 
+#[derive(Debug, PartialEq, Eq)]
+pub enum TraitItemType {
```

Comment:
```suggestion
pub enum TraitItem {
```

Reply:
Unfortunetly, this conflicts with `crates/noirc_frontend/src/ast/traits.rs`: pub enum TraitItem